sovtokenfees/sovtokenfees/test/helpers/helper_node.py

```diff
@@ -53,11 +57,11 @@ def get_fees_req_handler(self):
 
     def _reset_fees(self, node):
         req_handler = self._get_fees_req_handler(node)
-        empty_fees = req_handler.state_serializer.serialize({})
+        empty_fees = domain_state_serializer.serialize({})
         req_handler.state.set(build_path_for_set_fees().encode(), empty_fees)
 
     def _get_fees_req_handler(self, node):
-        return node.get_req_handler(ledger_id=CONFIG_LEDGER_ID)
+        return node.write_manager.request_handlers[NYM][-1]
```

Comment:
Please, use lookup through handler's list instead of indexing (isinstance(object, type))

Reply:
sure